app/static/tests/e2e/stochastic.etest.ts

```diff
@@ -22,18 +19,16 @@ test.describe("stochastic app", () => {
     });
 
     test("can change number of replicates and re-run model", async ({ page }) => {
+        await expect(await page.locator(".action-required-msg")).toHaveText("");
         await page.fill(":nth-match(#run-options input, 2)", "6");
 
-        await expect(await page.locator(".run-tab .action-required-msg")).toHaveText(
-            "Plot is out of date: number of replicates has changed. Run model to update.",
-            {
-                timeout
-            }
+        await expect(await page.locator(".action-required-msg")).toHaveText(
+            "Plot is out of date: number of replicates has changed. Run model to update."
```

Comment:
Feels like this little pair of checks - waiting for the action required element to be empty or to have particular text - appears often enough to merit helper methods.

Reply:
done